tests/python/pants_test/integration/log_output_integration_test.py

```diff
@@ -32,8 +32,8 @@ def test_completed_log_output(self) -> None:
             tmpdir_relative = self._prepare_sources(tmpdir, build_root)
 
             test_run_result = self.run_pants(
-                ["--no-dynamic-ui", "typecheck", f"{tmpdir_relative}/src/python/project"]
+                ["--no-dynamic-ui", "-ldebug", "typecheck", f"{tmpdir_relative}/src/python/project"]
             )
 
-            assert "[INFO] Starting: Run MyPy on" in test_run_result.stderr_data
-            assert "[INFO] Completed: Run MyPy on" in test_run_result.stderr_data
+            assert "[DEBUG] Starting: Run MyPy on" in test_run_result.stderr_data
+            assert "[DEBUG] Completed: Run MyPy on" in test_run_result.stderr_data
```

Comment:
Having these at DEBUG is odd I think... is that because it is skipped in this case? We should continue to render these, IMO.

Reply:
The `Process` is at DEBUG now because very quickly after finishing the Process, we will say 

```
17:45:43.76 [INFO] Completed: Typecheck using MyPy - succeeded.
Success: no issues found in 4 source files
```

It's noisy to have it be:

```
17:45:43.56 [INFO] Completed: Run MyPy on 4 files.
17:45:43.76 [INFO] Completed: Typecheck using MyPy - succeeded.
Success: no issues found in 4 source files
```